includes/data.php

```diff
@@ -1950,18 +1950,35 @@ function pods_serial_comma( $value, $field = null, $fields = null, $and = null,
 			$format = pods_v( 'repeatable_format', $params->field, 'default', true );
 
 			if ( 'default' !== $format ) {
-				$params->serial = false;
 
-				if ( 'custom' === $format ) {
-					$separator = pods_v( 'repeatable_format_separator', $params->field );
+				switch ( $format ) {
+					case 'ul':
+					case 'ol':
+						if ( is_array( $value ) ) {
+							$value = '<' . $format . '><li>' . implode( '</li><li>', $value ) . '</li></' . $format . '>';
```

Comment:
Formatting would be nice here

```suggestion
				$value = "
					<{$format}>
						<li>
							" . implode( "
								</li>
								<li>
								", $value ) . "
						</li>
					</{$format}>
				";
```

Reply:
I wouldn't do this. As mentioned in my other comment. White spaces and \n tags are very annoying when it comes to inline-block elements. And especially list type elements could be set as inline block for menu's etc.